<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tomeu Borras Riera</author>
 <class>BancoBase</class>
 <widget class="QWidget" name="BancoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bancos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../bulmages.qrc">
    <normaloff>:/Images/bank.png</normaloff>:/Images/bank.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../bulmages.qrc">:/Images/bank.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>5</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="3">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>291</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="mui_borrar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Borrar banco</string>
          </property>
          <property name="whatsThis">
           <string>Borrar banco</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:Images/edit-delete.png</normaloff>:Images/edit-delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="mui_guardar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Guardar banco</string>
          </property>
          <property name="whatsThis">
           <string>Guardar banco</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/edit-save.png</normaloff>:/Images/edit-save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="mui_nuevo">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Nuevo banco</string>
          </property>
          <property name="whatsThis">
           <string>Nuevo banco</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/document-new.png</normaloff>:/Images/document-new.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Lista de bancos:</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QListWidget" name="mui_lista">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Informacion del banco:</string>
        </property>
        <layout class="QGridLayout">
         <property name="margin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="11" column="2">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>D.C.:</string>
           </property>
           <property name="buddy">
            <cstring>mui_dcbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="11" column="3" colspan="3">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Numero de cuenta:</string>
           </property>
           <property name="buddy">
            <cstring>mui_numcuentabanco</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="5">
          <widget class="QLineEdit" name="mui_nombanco">
           <property name="maxLength">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="mui_cpbanco">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Fax:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_faxbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="4" colspan="2">
          <widget class="QLineEdit" name="mui_faxbanco">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="12" column="3" colspan="3">
          <widget class="QLineEdit" name="mui_numcuentabanco">
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="QLineEdit" name="mui_dcbanco">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLineEdit" name="mui_codagenciabanco">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLineEdit" name="mui_codentidadbanco">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="14" column="0" colspan="6">
          <widget class="QTextEdit" name="mui_comentbanco">
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>Razon Social:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_nombanco</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="mui_dirbanco">
           <property name="maxLength">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="5">
          <widget class="QLineEdit" name="mui_poblbanco">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maxLength">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="mui_telbanco">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="5">
          <widget class="QLineEdit" name="mui_emailbanco">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="5">
          <widget class="QLineEdit" name="mui_webbanco">
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="3">
          <widget class="QLineEdit" name="mui_contactobanco"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Contacto:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_contactobanco</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>URL:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_webbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>E-mail:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_emailbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Telefono:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_telbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Poblacion:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_poblbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Direccion</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_dirbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="6">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Comentarios:</string>
           </property>
           <property name="buddy">
            <cstring>mui_codentidadbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Entidad:</string>
           </property>
           <property name="buddy">
            <cstring>mui_codentidadbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Sucursal:</string>
           </property>
           <property name="buddy">
            <cstring>mui_codagenciabanco</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>C.P.:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_cpbanco</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Sufijo:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mui_sufijobanco</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QSpinBox" name="mui_sufijobanco">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="6">
          <widget class="QWidget" name="mui_plugin_frame" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>2</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2"/>
 <tabstops>
  <tabstop>mui_nombanco</tabstop>
  <tabstop>mui_dirbanco</tabstop>
  <tabstop>mui_cpbanco</tabstop>
  <tabstop>mui_poblbanco</tabstop>
  <tabstop>mui_telbanco</tabstop>
  <tabstop>mui_faxbanco</tabstop>
  <tabstop>mui_emailbanco</tabstop>
  <tabstop>mui_webbanco</tabstop>
  <tabstop>mui_contactobanco</tabstop>
  <tabstop>mui_codentidadbanco</tabstop>
  <tabstop>mui_codagenciabanco</tabstop>
  <tabstop>mui_dcbanco</tabstop>
  <tabstop>mui_numcuentabanco</tabstop>
  <tabstop>mui_comentbanco</tabstop>
  <tabstop>mui_nuevo</tabstop>
  <tabstop>mui_guardar</tabstop>
  <tabstop>mui_borrar</tabstop>
  <tabstop>mui_lista</tabstop>
 </tabstops>
 <resources>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
